src/components/UI/Tabs/Tabs.js

```diff
@@ -0,0 +1,35 @@
+import React, {Fragment} from 'react';
+import {useLocation} from 'react-router-dom';
+
+import {useTabsTranslation} from '../../../hooks';
+import {useApp} from '../../../providers/AppProvider';
+import {Divider} from '../Divider/Divider';
+import {Tab} from '../Tab/Tab';
+
+export const Tabs = () => {
+  const {navigateToRoute} = useApp();
+  const {termsTxt, faqTxt} = useTabsTranslation();
+  const {pathname} = useLocation();
+
+  const tabs = [
```

Comment:
To make this component reusable take the logic of this component to a new `<HeaderTabs /> and render `<Tabs />` from there

Reply:
Tabs are always part of the header. But indeed I will export the tabs array to a prop.